### PR TITLE
Anthos 1.1 Updates

### PR DESCRIPTION
**Pull request by meganokeefe**
- Updates cluster names from "central" and "remote" to  "gcp" and "onprem" (can change this back, if needed. this was specifically changed for Anthos Day NY) 
- Upgrades CSM Alpha --> **ASM Beta**  (with ASM Beta, no ASM-specific agents are installed on the GKE cluster. Uses stackdriver adapters instead.)
- Installs the GKE add-on for Istio (central cluster) instead of self-installed OSS Istio (in order to make Cloud Run on GKE work)
- Upgrades **ACM** to use Gatekeeper (admission control for policies) 
- Enables **Cloud Run** on GKE for the central cluster, and manually-installs knative on the onprem cluster, to get Cloud Run to work on both

### CODE REVIEW

**Comment by askmeegs** [comment]
@googlebot I consent

File: bootstrap-demo.sh [hunk 1]
```diff
@@ -43,56 +43,56 @@ if [[ $OSTYPE == "linux-gnu" && $CLOUD_SHELL == true ]]; then
...
         fi
-        
 
-        # Config repo source 
-    
-        read -e -p "Config Repo Source [https://github.com/cgrant/policy-repo]:" reposource 
+
+        # Config repo source
+
+        read -e -p "Config Repo Source [https://github.com/askmeegs/hipster]:" reposource
```
Comment (meganokeefe) [inline bootstrap-demo.sh]: TODO - change this to cgrant/policy repo once murog@'s PR lands 

https://github.com/cgrant/hipster

File: common/ql.profile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export GKE_CLUSTER=y
 export KOPS_GCE=y
-export GCE_CONTEXT=remote
+export GCE_CONTEXT=onprem
 export KOPS_AWS=n
-export REPO_URL=https://github.com/cgrant/policy-repo
+export REPO_URL=https://github.com/askmeegs/hipster
```
Comment (meganokeefe) [inline common/ql.profile]: TODO - switch to cgrant/hipster once murog@'s PR lands

File: config-management/create-repo.sh [hunk 3]
```diff
@@ -5,8 +5,8 @@ git config --global user.email "$EMAIL"
 git config --global user.name "$USER"
 
 export PROJECT=$(gcloud config get-value project)
-export REPO_URL=${REPO_URL:-"https://github.com/cgrant/hipster"}
-export REPO_BRANCH=${REPO_BRANCH:-"next2019"}
+export REPO_URL=${REPO_URL:-"https://github.com/askmeegs/hipster"}
```
Comment (meganokeefe) [inline config-management/create-repo.sh]: TODO revert to cgrant/hipster

File: config-management/install-config-sync.sh [hunk 4]
```diff
@@ -18,7 +18,7 @@
 
 
 export CLUSTER_NAME=$(kubectl config current-context)
-export REPO_URL=${REPO_URL:-"https://github.com/cgrant/policy-repo"}
+export REPO_URL="git@github.com:murog/hipster.git"
```
Comment (meganokeefe) [inline config-management/install-config-sync.sh]: TODO - switch to cgrant/hipster

**Pull request merged**